src/App/screens/Instructor/screens/LessonTopics/components/Submit/index.js

```diff
@@ -38,16 +70,16 @@ export default class Submit extends Component {
         </Heading>
 
         <div className='mb3'>
-          Got an idea for something to record? Great! Submit anything and everything. You can submit as many as you'd like; you can record it, or others can. We'll get back to you within a couple of days.
+          Got an idea for something to record? Great! Submit anything and everything. You can submit as many as you'd like. You'll automatically "Claim" your submissions.
```

Comment:
Changed this text from nick's wireframes to reflect what is happening currently.